tests/drivers/gpio/README.rst

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2025 Alif Semiconductor - All Rights Reserved.
+# Use, distribution and modification of this code is permitted under the
+# terms stated in the Alif Semiconductor Software License Agreement
+#
+# You should have received a copy of the Alif Semiconductor Software
+# License Agreement with this file. If not, please write to:
+# contact@alifsemi.com, or visit: https://alifsemi.com/license
+
+Connect P6_4 (J12_22) to P1_7 (J11_20)
+
+Supported Boards:
+	``alif_b1_dk/ab1c1f4m51820hh/rtss_he``
+	``alif_e8_dk/ae822fa0e5597xx0/rtss_he``
+	``alif_e8_dk/ae822fa0e5597xx0/rtss_hp``
+	``alif_e7_dk/ae722f80f55d5xx/rtss_he``
+	``alif_e7_dk/ae722f80f55d5xx/rtss_hp	``
```

Comment:
Please use a bulleted list.
* https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks